beacon-chain/sync/backfill/batch.go

```diff
@@ -139,3 +157,21 @@ func (b batch) withRetryableError(err error) batch {
 	b.err = err
 	return b.withState(batchErrRetryable)
 }
+
+func (b batch) blobsNeeded() int {
+	return b.bs.blobsNeeded()
+}
+
+func (b batch) blobResponseValidator() sync.BlobResponseValidation {
+	return b.bs.validateNext
+}
+
+func (b batch) availabilityStore() das.AvailabilityStore {
+	return b.bs.store
+}
+
+func sortBatchDesc(todo []batch) {
+	sort.Slice(todo, func(i, j int) bool {
+		return todo[j].end < todo[i].end
+	})
+}
```

Comment:
`todo` seems like a placeholder. Here's a suggestion, take it or leave it. I don't feel very strongly about this.

```suggestion
func sortBatchDesc(bb []batch) {
	sort.Slice(bb, func(i, j int) bool {
		return bb[j].end < bb[i].end
	})
}
```

Reply:
ah the list of batches is called `todo` elsewhere. I should rename it later to avoid confusion (also annoying because the linter perks up when it's mentioned in a comment). Will take this suggestion though as its self-contained.